app/assets/stylesheets/application.css.scss

```diff
@@ -3,6 +3,9 @@
 @import 'required';
 @import 'vendor';
 @import 'vendor-overrides';
```

Comment:
Can we delete this file, since as noted in the comment, it existed only as long this ticket wasn't yet implemented.

https://github.com/18F/identity-idp/blob/7c9ae428004c73cca870f894b3c04e05b7777572/app/assets/stylesheets/_vendor-overrides.scss#L1-L5